scripts/runhelpers.cake

```diff
@@ -151,6 +151,37 @@ ExitStatus Run(string exec, string args, RunOptions runOptions)
 }
 
 /// <summary>
+///  Run restore with the given arguments
+/// </summary>
+/// <param name="exec">Executable to run</param>
+/// <param name="args">Arguments</param>
+/// <param name="runOptions">Optional settings</param>
+/// <returns>The exit status for further queries</returns>
+ExitStatus RunRestore(string exec, string args, string workingDirectory)
+{
```

Comment:
I would refactor this to use Run internally to keep with the uniformity.

`var restoreLog = new List<string>();`
`var exitCode = Run(exec, args, new RunOptions`
 `                   {`
`                        WorkingDirectory = workingDirectory,`
`                        StandardOutputListing = restoreLog`
`                    });`
`if (exitCode != 0)`
`{`
`    Error(string.Join("\n", restoreLog));`
`}`
`return exitCode;`